src/main/java/io/muserver/murp/ReverseProxyBuilder.java

```diff
@@ -184,23 +186,29 @@ public static ReverseProxyBuilder reverseProxy() {
     /**
      * Creates a new ReverseProxy which is a MuHandler. You can pass the resulting handler directly
      * to {@link io.muserver.MuServerBuilder#addHandler(MuHandler)}
+     *
      * @return A MuHandler that acts as a reverse proxy
      */
     @Override
     public ReverseProxy build() {
         if (uriMapper == null) {
             throw new IllegalStateException("A URI mapper must be specified");
         }
+
         HttpClient client = httpClient;
         if (client == null) {
-            client = HttpClientBuilder.httpClient().build();
+            client = HttpClientUtils.createHttpClientBuilder(true)
```

Comment:
Always go secure-by-default.

This probably means you can delete the trust-all client from the source code (or maybe move it to the test source for testing). If people want to not trust certs then they'll need to put in the extra effort to achieve that.

Reply:
will add ReverseProxyBuilder.createHttpCleint() as below, so that we can pass the trust all option to api user. 

`
    /**
     * Creates a new HTTP Client builder that is suitable for use in mu reverse proxy.
     *
     * @param trustAll If true, then any SSL certificate is allowed.
     * @return An HTTP Client builder
     */
    public static HttpClient.Builder createHttpClient(boolean trustAll) {
        return createHttpClientBuilder(trustAll)
            .followRedirects(HttpClient.Redirect.NEVER);
    }
`